pkg/webhook/admission/webhook.go

```diff
@@ -203,3 +208,62 @@ func (w *Webhook) InjectFunc(f inject.Func) error {
 
 	return setFields(w.Handler)
 }
+
+// InstrumentedHook adds some instrumentation on top of the given webhook.
+func InstrumentedHook(path string, hookRaw http.Handler) http.Handler {
```

Comment:
I moved `InstrumentedHook` from server.go here into `admission/webhook.go` because I'm using it in `StandaloneWebhook` but also because it seemed more appropriate because its wrapping instrumentation around the handler rather than the server stuff. Let me know if you think somewhere else is more appropriate.

Reply:
this applies to all webhooks, not just admission ones, so it belongs in `pkg/webhook`.

Reply:
or you could put it in a subpackage to avoid import loops, if necessary, like `pkg/webhook/internal/metrics` or something